source/shared/cpp/ObjectModel/TextBlock.cpp

```diff
@@ -86,7 +89,7 @@ Json::Value TextBlock::SerializeToJsonValue()
 
 std::string TextBlock::GetText() const
 {
-    return m_text;
+    return ParseISO8601UsingRegex();
```

Comment:
>ParseISO8601UsingRegex [](start = 11, length = 22)

UsingRegex suffix is unnecessary

Reply:
Agree

---
In reply to: [148858644](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r148858644) [](ancestors = 148858644)

Reply:
Still there :)

---
In reply to: [148859669](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r148859669) [](ancestors = 148859669,148858644)

Reply:
Thanks~

---
In reply to: [149210701](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r149210701) [](ancestors = 149210701,148859669,148858644)